org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/launch/npm/NpmLaunchShortcut.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2019 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Mickael Istria (Red Hat Inc.) - initial implementation
```

Comment:
Update contributors

Reply:
Or even get rid of the contributors area. It's not required, so we can live without it. it's less maintenance.

Reply:
Agreed, done.